src/store/asana/index.ts

```diff
@@ -146,6 +152,12 @@ export default {
       state.projects = sortAndUnique(state.projects);
       jsonstore.set("projects", state.projects);
     },
+    addAllTags(state: State, payload: TaskTag[]): void {
```

Comment:
I think we need to use some different words. "add" is used elsewhere to signify it will be added to a task. Here, it seems to be for local state. Maybe change this to "set"?